### PR TITLE
Update vector.md: wording

### PR DESCRIPTION
**Pull request by gittywithexcitement**

### CODE REVIEW

File: content/vector.md [hunk 1]
```diff
@@ -262,7 +263,7 @@ prefix your function calls with `V.`.
 
 * Exercise 3: Use an unboxed (or storable) vector instead of the boxed vectors
   we were using above. What code did you have to change from the original
-  example?  Do your examples from exercise 2 all work still?
+  example?  Do your all of your examples from exercise 2 still work?
```
Comment (snoyberg) [inline content/vector.md]: I think there's an extra "your" here

**Comment by snoyberg** [comment]
Looks great, I just had one comment.

**Comment by gittywithexcitement** [comment]
Good catch, fixed.

**Comment by snoyberg** [comment]
Thanks!

**Pull request merged**